### PR TITLE
login screen

### PR DESCRIPTION
**Pull request by potetz**

### CODE REVIEW

**Comment by potetz** [comment]
[Create landing page](https://trello.com/c/A88SvRgy/28-create-landing-page)

File: database/templates/database/.gitkeep [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (daniddelrio) [inline database/templates/database/.gitkeep]: Don't commit this file! I'm not sure what this file does HAHA

File: tanong_db/urls.py [hunk 2]
```diff
@@ -1,7 +1,6 @@
-from django.contrib import admin
-from django.urls import include, path
-
+from django.urls import path
+from database import views
 urlpatterns = [
-    path('admin/', admin.site.urls),
-    path('', include('database.urls'))
+    # path('', )
+    path('login/', views.login)
```
Comment (daniddelrio) [inline tanong_db/urls.py]: Don't remove the first two lines as these are meant to be in the `urls.py` of the main project.

`path('login/', views.login)` should be in the `urls.py` of our app folder, which is `database`, and not the urls.py of `tanong_db`.

File: tanong_db/views.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import render
+
+# Create your views here.
+def login(request):
+ return render(request, 'database/login.html')
```
Comment (daniddelrio) [inline tanong_db/views.py]: This should also be in the `views.py` under `database`, not `tanong_db`! Typically the main project folder shouldn't contain a `views.py` file hehe

**Review by daniddelrio — CHANGES_REQUESTED** [review]
Nice to see you got the hang of Django even just a little bit! And good work also on the landing page; it looks neat :)) A few general comments lang:

1. Inline CSS is fine but I personally prefer using CSS files instead. I realized I forgot to teach you how to use CSS files in the Django environment, but I pushed some files to your branch to make things easier for you. Just add your CSS to database/static/database/css/main.css. If you wanna know how I set up the CSS, check my commit hehe
2. Maybe you can try using `form-control-sm` instead of `form-control` as I think the fields are too big? Or maybe adjust the padding of the fields by using [Bootstrap's built-in spacing utility classes](https://getbootstrap.com/docs/4.4/utilities/spacing/).
3. Use Roboto font like screens did! Just edit the main.css.

**Review by daniddelrio — APPROVED** [review]
Looks good to me! Comments lang for your future work:
1. Try to make your commit messages more informative so it'll easier to track the changes. [Read here](https://chris.beams.io/posts/git-commit/) on how to make good commit messages.
2. I recommend making the CSS for classes instead of IDs to increase reusability :)

**Pull request merged**